### PR TITLE
Adds the CP33

### PR DESCRIPTION
**Pull request by Holli-Git**
#### Summary
Content "Adds the CP33"

#### Purpose of change

The CP33 is a funky lil gun that's really cool. The "large capacity handgun" niche is also only really filled in by the glock, and that feels boring. So, here's the solution!

#### Describe the solution

Adds the funky lil gun that's really cool.

#### Describe alternatives you've considered

Not adding the funky lil gun that looks really cool.

#### Testing

Tested locally, works fine!

#### Additional context
image
Just look at this lil goober. 33 rounds? You gotta be kidding me. I <3 Keltec

cp33

### CODE REVIEW

File: data/json/items/gun/22.json [hunk 1]
```diff
@@ -407,5 +407,46 @@
     "weapon_category": [ "AUTOMATIC_PISTOLS" ],
     "pocket_data": [ { "magazine_well": "126 ml", "pocket_type": "MAGAZINE_WELL", "item_restriction": [ "wp22mag" ] } ],
     "melee_damage": { "bash": 4 }
+  },
+  {
+    "id": "cp33",
+    "copy-from": "pistol_base",
+    "looks_like": "ar_pistol",
+    "type": "GUN",
+    "name": { "str": "high capacity .22 handgun" },
```
Comment (anothersimulacrum) [inline data/json/items/gun/22.json:416]: Ammotypes are automatically added to names by default
```suggestion
    "name": { "str": "high capacity handgun" },
```

File: data/json/items/magazine/22.json [hunk 2]
```diff
@@ -333,5 +333,57 @@
     "ammo_type": [ "22" ],
     "flags": [ "MAG_COMPACT" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "22": 10 } } ]
+  },
+  {
+    "id": "cp33_mag",
+    "looks_like": "glock17_17",
+    "type": "MAGAZINE",
+    "name": { "str": "high capacity 33 round .22 magazine" },
```
Comment (anothersimulacrum) [inline data/json/items/magazine/22.json:341]: These can also drop the .22, but I don't feel like it's as redundant here.

File: data/json/items/gun/22.json (hunk 1 above)
Comment (Holli-Git) [reply data/json/items/gun/22.json:416]: Ah, thanks!

**Comment by Maleclypse** [comment]
Can you add a screenshot of where you found enough for sale to qualify for inclusion.  I've been wanting to make that a standard requirement for any new guns so that we don't have someone coming back six months from now and going "I can't find x number of this gun for sale and therefore it never should have been included".

**Pull request merged**